relay-dynamic-config/src/global.rs

```diff
@@ -23,6 +25,9 @@ pub struct GlobalConfig {
     /// Quotas that apply to all projects.
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub quotas: Vec<Quota>,
+    /// Configuration for global inbound filters.
+    #[serde(skip_serializing_if = "skip_default_error_boundary")]
+    pub filters: ErrorBoundary<GenericFiltersConfig>,
```

Comment:
it seems similar to `MeasurementsConfig` in it's form. So maybe we should also add errorboundary to the measurementsconfig in another PR if it makes sense to do it in genericfiltersconfig 🤔

Reply:
What's the uncovered use case that adding `ErrorBoundary` to `MeasurementsConfig` covers?